javatests/arcs/showcase/typevariables/variable.arcs

```diff
@@ -18,15 +18,13 @@ particle OrderIngestion in '.OrderIngestion'
 particle SkuRedactor in '.SkuRedactor'
   input: reads [~a with {sku: Text}]
   output: writes [~a]
-  // TODO(b/160694452) support proper claims and checks
-  // claim output.sku is redacted
+  claim output.sku is redacted
```

Comment:
Here is an "integration" test of sorts: Try running the codegen here: 
`bazel build //javatests/arcs/showcase/typevariables:var_generation`